### ISSUE TITLE
Errors on all GF API conditions

### ISSUE THREAD

**Issue by simoncozens** [googlefonts/fontbakery#3675]
(**NOTE:** If your issue is related to a specific check, please consider mentioning its check-id on the issue title.)

## Observed behaviour

Today I'm getting JSON-related errors on anything that makes a call to the GF API. Did an endpoint change?

💔 <b>ERROR:</b> Check METADATA.pb includes production subsets.<div>
* [com.google.fonts/check/metadata/includes_production_subsets](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/metadata/includes_production_subsets)

* 💔 **ERROR** The condition <FontBakeryCondition:production_metadata> had an error: JSONDecodeError: Expecting value: line 1 column 1 (char 0)
</div>💔 <b>ERROR:</b> Version number has increased since previous release on Google Fonts?<div>

* [com.google.fonts/check/version_bump](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/version_bump)

* 💔 **ERROR** The condition <FontBakeryCondition:api_gfonts_ttFont> had an error: FailedConditionError: The condition <FontBakeryCondition:remote_styles> had an error: JSONDecodeError: Expecting value: line 1 column 1 (char 0)
</div>💔 <b>ERROR:</b> Glyphs are similiar to Google Fonts version?<div>

* [com.google.fonts/check/production_glyphs_similarity](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/production_glyphs_similarity)

* 💔 **ERROR** The condition <FontBakeryCondition:api_gfonts_ttFont> had an error: FailedConditionError: The condition <FontBakeryCondition:remote_styles> had an error: JSONDecodeError: Expecting value: line 1 column 1 (char 0)
</div>💔 <b>ERROR:</b> Check if the vertical metrics of a family are similar to the same family hosted on Google Fonts.<div>

* [com.google.fonts/check/vertical_metrics_regressions](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/vertical_metrics_regressions)

* 💔 **ERROR** The condition <FontBakeryCondition:regular_remote_style> had an error: FailedConditionError: The condition <FontBakeryCondition:remote_styles> had an error: JSONDecodeError: Expecting value: line 1 column 1 (char 0)
</div>💔 <b>ERROR:</b> Check font follows the Google Fonts CJK vertical metric schema<div>

* [com.google.fonts/check/cjk_vertical_metrics](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/cjk_vertical_metrics)

* 💔 **ERROR** The condition <FontBakeryCondition:remote_styles> had an error: JSONDecodeError: Expecting value: line 1 column 1 (char 0)
</div>💔 <b>ERROR:</b> Check if the vertical metrics of a CJK family are similar to the same family hosted on Google Fonts.<div>

* [com.google.fonts/check/cjk_vertical_metrics_regressions](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/cjk_vertical_metrics_regressions)

* 💔 **ERROR** The condition <FontBakeryCondition:regular_remote_style> had an error: FailedConditionError: The condition <FontBakeryCondition:remote_styles> had an error: JSONDecodeError: Expecting value: line 1 column 1 (char 0)
</div>

## Expected behaviour

No errors, or a better error message.

**Comment by marekjez86** [comment]
GitHub was down for a while. All GitHub. Would it cause this issue, if you
couldn't reach it?

**Comment by simoncozens** [comment]
No, it's a GF API check. The problem is that:

* production_metadata has no error checking! Bad programmer!
* The metadata is now valid JSON, so the lines:

```
    # can't do requests.get("url").json() since request text starts with ")]}'"
    return json.loads(requests.get(meta_url).text[5:])
```

are no longer true.

**Comment by simoncozens** [comment]
Looks like this was addressed in ac64a301.
